stac_fastapi/opensearch/pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
     "opensearch-py[async]~=2.8.0",
     "uvicorn~=0.23.0",
     "starlette>=0.35.0,<0.36.0",
+    "redis==6.4.0",
```

Comment:
Redis should not be installed in the core package as most Users probably won't use Redis. It can be installed ie. stac-fastapi-elasticsearch[redis] or with dev